src/sentry/api/bases/organization_events.py

```diff
@@ -484,7 +490,7 @@ def get_event_stats_data(
             # there were no top events found. In this case, result contains a zerofilled series
             # that acts as a placeholder.
             is_multiple_axis = len(query_columns) > 1
-            if top_events > 0 and isinstance(result, dict):
+            if isinstance(result, dict):
```

Comment:
Why remove the `top_events` check?

Reply:
as far as I can tell the two imply each other -- and the other two branches will always crash if given the dict half of the union